awscli/help.py

```diff
@@ -59,12 +60,22 @@ class HelpRenderer(object):
     """
 
     def render(self, contents):
+        output = self.format(contents)
+        self.get_pager_cmdline()
+        pydoc.pager(output)
+        sys.exit(1)
+    
+    def format(self, contents):
         """
         Each implementation of HelpRenderer must implement this
-        render method.
+        format method unless it implements render.
         """
         pass
 
+    def get_pager_cmdline(self):
```

Comment:
It seems odd to have a method called `get_pager_cmdline` that doesn't actually return anything.